tools/shared.py

```diff
@@ -94,6 +94,8 @@ def shlex_quote(arg):
 # Switch to shlex.join once we can depend on python 3.8:
 # https://docs.python.org/3/library/shlex.html#shlex.join
 def shlex_join(cmd):
+  if type(cmd) is str:
+    return cmd
```

Comment:
Is this related?

Reply:
Only in the sense that SIMD tests are the only ones that pass strings to `run_process`.  e.g. they do `shared.run_process('thing')`.   I hadn't realized that was valid... but when we print the string version of the process (e.g. in --verbose mode) it was showing up as `t h i n g` :)